### ISSUE TITLE
Question: Porting under the same license to other games

### ISSUE THREAD

**Issue by Anonymous-BCFED** [VRCFury/VRCFury#293]
Under the current dual-licensing arrangement, my interpretation is that it would be permissible to fork this repository under the same license, *with the same attributions and without requiring payment* (which are the requirements for CC BY-NC 4.0), so that users who wish to use the features of this library could use it on other social platforms.

Is this a correct interpretation of your team's wishes? Asking for a friend, who is nervous about it.

**Comment by SenkyDragon** [comment]
That sounds correct.

Beware that even distribution without directly requiring payment for the fork itself may still be considered as commercial use if it is being used primarily as a value-add to a product that otherwise does solicit money. Some examples of a potential violation may be:
* Distributing a fork of VRCFury and soliciting donations or subscriptions for that distrubution
* Distributing a fork of VRCFury primarily for the purpose of enabling or enhancing a product that costs money, solicits donations or subscriptions

**Comment by Anonymous-BCFED** [comment]
Nah, he just wants to open up the VRCFury ecosystem to CVR.  From your response, it sounds like you won't have a problem with that as long as he differentiates trademarks.
